### ISSUE TITLE
Error when trying to update existing declaration with self-ip

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-declarative-onboarding#135]
### Environment
 * Declarative Onboarding Version: 1.11.0
 * BIG-IP Version: 15.0.1

### Summary
DO error trying to change port lockdown on existing declaration - DO 1.11.0, BIG-IP 15.0.1.1:

error = 01071412:3: Cannot delete IP (10.0.2.4) because it is used by the system config-sync setting

My setup:
Trying to setup HA. I forgot to set external NIC to port lockdown default. Initial declaration had none, so hence HA failed. I tried to push declaration again with default, but I received error above.

### Steps To Reproduce
1. Submit the following declaration:
```
	"external-localself": {
	"class": "SelfIp",
	"address": "10.0.2.4/24",
	"vlan": "external",
	"allowService": "none", <<<<<<< first declare set to "none"
	"trafficGroup": "traffic-group-local-only"
```

2. Observe the following error response:
Received 200 OK, but I realized that allowService was set to "none" so HA didn't form. I then updated my declaration snippet of the network section from "none" to "default" like this and re-POSTed...
```
	"external-localself": {
	"class": "SelfIp",
	"address": "10.0.2.4/24",
	"vlan": "external",
	"allowService": "default", <<< tried to change 2nd declare to "default"
	"trafficGroup": "traffic-group-local-only"
```

### Expected Behavior
Declaration was identical with the exception of "allowService" changing from none to default. DO error'ed though. I expected the declaration to simply update the external-localself to allow "default".

### Actual Behavior
error = 01071412:3: Cannot delete IP (10.0.2.4) because it is used by the system config-sync setting

### Workaround
Workaround = manually changed port lockdown in GUI to "default". Then re-pushed declaration with default...works.

**Comment by dstokesf5** [comment]
@JeffGiroux Thank you for reporting this. I was able to reproduce it and I have added it to our internal product backlog as AUTOTOOL-1797.

**Comment by erjac77** [comment]
Hi, I have a similar, but slightly different problem. I send the exact same declaration (so no change to the self-ip), but it gives this error:

`010713b1:3: Cannot delete IP (x.x.x.x) because it is used by the system state-mirroring (Primary Mirror Address) setting.`

DO 1.15.0
BIG-IP 13.1.1.4 Build 0.0.4 Point Release 4

Please confirm if this is the same problem or if I need to create a new one.

**Comment by dstokesf5** [comment]
@erjac77 The two issues seem closely related, and a single fix will likely handle both. However, there is enough difference in the steps to reproduce the issue that a new issue would be prudent.

**Comment by dstokesf5** [comment]
This has been resolved in the 1.26.0 release.
